pygpu/tests/test_gpu_ndarray.py

```diff
@@ -41,15 +41,16 @@ def test_hash():
     g = pygpu.empty((2, 3), context=ctx)
     exc = None
     try:
-        h = hash(g)
+        hash(g)
     except TypeError as e:
         exc = e
     assert exc is not None
 
 
 def test_bool():
     for data in [numpy.empty((0, 33)), [[1]], [[0]], [], [0], [1], 0, 1]:
-        assert bool(pygpu.asarray(data, context=ctx)) == bool(numpy.asarray(data))
+        assert bool(pygpu.asarray(data, context=ctx)) == \
```

Comment:
We try to remove that syntax (\ at the end of a line) in Theano. We use paranthesis instead. I think it is more readable and other agreed on that. Why do you do differently?